src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs

```diff
@@ -2107,7 +2152,35 @@ private object DeserializeFromTransformer(Type entityType, string id, BlittableJ
             HandleInternalMetadata(document);
             return JsonConverter.FromBlittable(entityType, ref document, id, trackEntity);
         }
+        
+        internal bool CheckIfChangeVectorAlreadyIncluded(IEnumerable<string> changeVectors)
+        {
+            if (IncludeRevisionsByChangeVector is null) 
+                 return false;
+            
+            foreach (var cv in changeVectors)
+            {
+                if (IncludeRevisionsByChangeVector.TryGetValue(cv, out DocumentInfo documentInfo) == false )
+                    return false;
+                
+                if ((documentInfo.Entity == null) && (documentInfo.Document == null))
+                    return false;
```

Comment:
This logic seems suspicious. shouldn't you return `true` here when the value is found?